libraries/RW/CLI/CLI.py

```diff
@@ -119,6 +118,36 @@ def run_bash_file(
     cmd_overide: str = "",
     **kwargs,
 ) -> platform.ShellServiceResponse:
+
+    # Check if the file exists in the current working directory
+    if os.path.exists(bash_file):
+        logger.info(f"File '{bash_file}' found in the current working directory.")
+    else:
+        cwd = os.getcwd()
+
+        # Check if the current working directory is the root
+        if cwd == "/":
+            # Check if RW_PATH_TO_ROBOT environment variable exists
+            rw_path_to_robot = os.environ.get('RW_PATH_TO_ROBOT', None)
```

Comment:
I can poke at it post-merge if you're short on time for the demo

I'm inclined to try:

```
from robot.libraries.BuiltIn import BuiltIn
dir = BuiltIn().get_variable_value("${CURDIR}")
```

Reply:
Totally up to you